distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -2958,9 +2958,10 @@ Table.from (that : Text) (format:Delimited_Format = Delimited_Format.Delimited '
    Arguments:
    - that: The table to convert.
    - format: The format of the text.
-Text.from (that : Table) (format:Delimited_Format = Delimited_Format.Delimited '\t') = case format of
-    _ : Delimited_Format -> Delimited_Writer.write_text that format
-    _ -> Unimplemented.throw "Text.from is currently only implemented for Delimited_Format."
+Text.from (that : Table) (format:Delimited_Format = Delimited_Format.Delimited '\t') =
+    File_Format.handle_format_missing_arguments format <| case format of
+        _ : Delimited_Format -> Delimited_Writer.write_text that format
+        _ -> Unimplemented.throw "Text.from is currently only implemented for Delimited_Format."
```

Comment:
not needed here as the `format` argument is type checked.
We possibly should remove some of the code from here or the type checking.

Reply:
Oh indeed I have missed that.

Hmm, I guess we can keep the typecheck and simplify.